pinc/project_quick_check.inc

```diff
@@ -450,6 +449,12 @@ function _test_project_for_bad_bytes($projectid)
 
     while ([$text, $imagename, $proofers] = page_info_fetch($pages_res)) {
         $round_num = get_round_num_from_proofers($proofers);
+        if ($round_num == 0) {
```

Comment:
I guess it doesn't make sense to add a proper Round for OCR here https://github.com/DistributedProofreaders/dproofreaders/blob/d9e338bacfa4ed8a6547bbafd54dc4b88742cfac/pinc/stages.inc so we could use get_Round_for_round_number like our other rounds for various reasons, but just saying it in case it has just been missed.

Reply:
Good question. Unfortunately the code treats Stages as "things that can be done" and a lot of our code does various things to iterate over them, like the Activity Hub and (IIRC) the navigation header. I think we'd need to do a whole lot of other special casing to make OCR a stage of some sort. Something worth thinking about as we do this "OCR is a sort of stage thing" allllllll over the place.